tests/integration/hive_metastore/test_migrate.py

```diff
@@ -19,12 +23,45 @@
 
 from ..conftest import (
     StaticGrantsCrawler,
+    StaticMountCrawler,
     StaticTableMapping,
     StaticTablesCrawler,
     StaticUdfsCrawler,
 )
 
 logger = logging.getLogger(__name__)
+_SPARK_CONF = {
+    "spark.databricks.cluster.profile": "singleNode",
+    "spark.master": "local[*]",
+    "fs.azure.account.auth.type.labsazurethings.dfs.core.windows.net": "OAuth",
+    "fs.azure.account.oauth.provider.type.labsazurethings.dfs.core.windows.net": "org.apache.hadoop.fs"
+    ".azurebfs.oauth2.ClientCredsTokenProvider",
+    "fs.azure.account.oauth2.client.id.labsazurethings.dfs.core.windows.net": "dummy_application_id",
+    "fs.azure.account.oauth2.client.secret.labsazurethings.dfs.core.windows.net": "dummy",
+    "fs.azure.account.oauth2.client.endpoint.labsazurethings.dfs.core.windows.net": "https://login"
+    ".microsoftonline.com/directory_12345/oauth2/token",
+}
+
+
+def principal_acl(ws, inventory_schema, sql_backend):
+    installation = MockInstallation(
```

Comment:
You need to stop using the MockInstallation for integration test to work properly: Workflow is reading the CSV file and can’t find it on a workspace and fails. You can find "real installation" instance as one of the fields returned by new_installation() fixture